packages/medusa-js/src/resources/admin/sales-channels.ts

```diff
@@ -1,10 +1,12 @@
 import {
+  AdminPostSalesChannelReq,
```

Comment:
todo: pluralised, based on the convention, I think this should be pluralised to follow the path on which we are doing the request. `AdminPostSalesChannelsReq`